flashinfer/experimental/sm12x/attention/_shared/contiguous/api.py

```
@@ -0,0 +1,2548 @@
+    "VarlenAttentionBinding",
+    "VarlenAttentionPlan",
+    "VarlenAttentionPlanKey",
+    "VarlenAttentionScratchPlan",
+    "sm12x_attention_forward",
+    "sm12x_varlen_attention_forward",
+    "clear_attention_caches",
+    "create_attention_plan",
+    "create_varlen_attention_plan",
+    "plan_attention_scratch",
+    "plan_varlen_attention_scratch",
+]
```

Ruff (0.15.21)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)